langchain4j-open-ai-official/src/test/java/dev/langchain4j/model/openaiofficial/InternalOpenAiOfficialTestHelper.java

```diff
@@ -0,0 +1,309 @@
+                    .azureApiKey(System.getenv("AZURE_OPENAI_KEY"))
+                    .modelName(CHAT_MODEL_NAME)
+                    .build();
+
+            AZURE_OPEN_AI_CHAT_MODEL_STREAMING_WITH_STRICT_TOOLS = OpenAiOfficialStreamingChatModel.builder()
+                    .baseUrl(System.getenv("AZURE_OPENAI_ENDPOINT"))
+                    .azureApiKey(System.getenv("AZURE_OPENAI_KEY"))
+                    .modelName(CHAT_MODEL_NAME)
+                    .strictTools(true)
+                    .build();
+
+            AZURE_OPEN_AI_STREAMING_CHAT_MODEL_JSON_WITH_STRICT_SCHEMA = OpenAiOfficialStreamingChatModel.builder()
+                    .baseUrl(System.getenv("AZURE_OPENAI_ENDPOINT"))
+                    .azureApiKey(System.getenv("AZURE_OPENAI_KEY"))
+                    .modelName(CHAT_MODEL_NAME)
+                    .supportedCapabilities(Set.of(RESPONSE_FORMAT_JSON_SCHEMA))
+                    .strictJsonSchema(true)
+                    .build();
+
+            AZURE_OPEN_AI_EMBEDDING_MODEL = OpenAiOfficialEmbeddingModel.builder()
+                    .baseUrl(System.getenv("AZURE_OPENAI_ENDPOINT"))
+                    .azureApiKey(System.getenv("AZURE_OPENAI_KEY"))
+                    .modelName(EMBEDDING_MODEL_NAME)
+                    .build();
+
+            AZURE_OPEN_AI_IMAGE_MODEL = OpenAiOfficialImageModel.builder()
+                    .baseUrl(System.getenv("AZURE_OPENAI_ENDPOINT"))
+                    .azureApiKey(System.getenv("AZURE_OPENAI_KEY"))
+                    .size(ImageGenerateParams.Size._1024X1024)
+                    .modelName(IMAGE_MODEL_NAME)
+                    .build();
+
+            AZURE_OPEN_AI_IMAGE_MODEL_BASE64 = OpenAiOfficialImageModel.builder()
+                    .baseUrl(System.getenv("AZURE_OPENAI_ENDPOINT"))
+                    .azureApiKey(System.getenv("AZURE_OPENAI_KEY"))
+                    .modelName(IMAGE_MODEL_NAME)
+                    .responseFormat(ImageGenerateParams.ResponseFormat.B64_JSON)
+                    .build();
+
+        } else {
+            AZURE_OPEN_AI_CHAT_MODEL = null;
+            AZURE_OPEN_AI_CHAT_MODEL_WITH_STRICT_TOOLS = null;
+            AZURE_OPEN_AI_CHAT_MODEL_JSON_WITHOUT_STRICT_SCHEMA = null;
+            AZURE_OPEN_AI_CHAT_MODEL_JSON_WITH_STRICT_SCHEMA = null;
+            AZURE_OPEN_AI_STREAMING_CHAT_MODEL = null;
+            AZURE_OPEN_AI_CHAT_MODEL_STREAMING_WITH_STRICT_TOOLS = null;
+            AZURE_OPEN_AI_STREAMING_CHAT_MODEL_JSON_WITH_STRICT_SCHEMA = null;
+            AZURE_OPEN_AI_EMBEDDING_MODEL = null;
+            AZURE_OPEN_AI_IMAGE_MODEL = null;
+            AZURE_OPEN_AI_IMAGE_MODEL_BASE64 = null;
+        }
+
+        // Set up OpenAI models if the environment variables are set
+        if (System.getenv("OPENAI_API_KEY") != null) {
+            OPEN_AI_CHAT_MODEL = OpenAiOfficialChatModel.builder()
+                    .apiKey(System.getenv("OPENAI_KEY"))
```

Comment:
```suggestion
                    .apiKey(System.getenv("OPENAI_API_KEY"))
```
Please change it everywhere, as otherwise it will fail (at leas on my env). I tried to push this change myself, but I do not have permissions

Reply:
OK

Reply:
OK